README.md

```diff
@@ -158,6 +158,29 @@ dnf install golang-bin git
 You should now have everything needed to [try out](#basic-usage) the CLI or hack on the code in ~~vim~~ your favorite
 editor.
 
+### Implementing new package managers
+
+One of Cachi2's core goals is to be a kind of package manager wrapper (around
+e.g. gomod, pip, npm...)
+
+Given the multitude of languages and therefore package managers, contributions
+are **definitely** encouraged.
+
+**Testing**: To ease development, we've implemented a flag
```

Comment:
I'd personally be more technical approaching this, i.e. use less text/sentences and just provide a brief explanantion a more importantly list of CLI flags we provide for easy testing of in development stuff, because there is a chance we'll add more `dev-` hidden flags in the future. IOW make this section more developer-oriented, people who are supposed to be able to navigate through *any* code if they decide to contribute to a project and just instruct them what primitives we prepared for them to make their lives as contributors easier rather than let them spend time reading normal sentences.